guides/doc-Managing_Configurations_Puppet/master.adoc

```diff
@@ -0,0 +1,17 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: managing-configurations-puppet
+
+= Managing Configurations Using Puppet Integration in {Project}
```

Comment:
I somewhat dislike the title.

What about `Configuring Hosts Using Puppet`? I think its both shorter and more precise.

Also, it would give us room to create two more guides like `... using Ansible` and `... using Salt`.

Reply:
I think @maximiliankolb is onto something here. I agree.

Reply:
1. Good point with the various guides, I was wondering about the same
2. We would like to have product name in the main title -- can we have two titles conditioned per product?
3. second commit -- I'll elaborate in the commit message

Reply:
We can have upstream and downstream titles so if Shweta doesn't like the version that is consistent with upstream, let's make two.

Reply:
Anyway, I'm going to use conditional titles either way, because we still want to include the product name.

Reply:
But what we need to decide for the upstream is the guide URL and I'm presuming that we prefer it in the same wording as the actual title?

Reply:
Avoiding titles where possible in the URL is the best approach. It makes it vendor neutral and this is our desired end state.

Reply:
@melcorr I've added Max's title and could we perhaps merge this now, please?

Reply:
Your wish is my command.